requirements.txt

```diff
@@ -1,11 +1,7 @@
 praw==4.4.0
 redis<3.0.0
-hiredis
```

Comment:
I didn't notice it imported anywhere. Figured it was a dependency of a dependency or something.